service/tasks/driver.py

```diff
@@ -1132,13 +1131,12 @@ def _deploy_instance(driverCls, provider, identity, instance_id,
         _deploy_instance.retry(exc=exc)
     except LibcloudDeploymentError as exc:
         celery_logger.exception(exc)
```

Comment:
here the celery logs have the entire exception logged. so there is no need to really do anything else.